Sources/PortalSwift/Core/Mpc/PortalMpc.swift

```diff
@@ -203,25 +217,87 @@ public class PortalMpc {
 
   public func eject(
     _ method: BackupMethods,
-    withCipherText: String,
-    andOrganizationBackupShare: String,
+    withCipherText: String? = nil,
+    andOrganizationBackupShare: String? = nil,
+    andOrganizationSolanaBackupShare: String? = nil,
     usingProgressCallback _: ((MpcStatus) -> Void)? = nil
-  ) async throws -> String {
+  ) async throws -> [PortalNamespace: String] {
```

Comment:
does this change in the return type make it a breaking change?

Reply:
No, because the `Portal` class's function handles this.

Reply:
I'm not sure, but maybe we need to change the `PortalMpc` class functions to be `internal` instead of `public` at least the ones wrapped by `Portal` class

Reply:
Yeah, probably. I believe the `mpc` property of the `Portal` class is private now, though, so they'd have to create their own instance to use these functions.